contrib/system_containers/centos/run.sh

```diff
@@ -8,4 +8,4 @@ printf %s $LABEL > /proc/self/attr/exec
 test -e /etc/sysconfig/crio-storage && source /etc/sysconfig/crio-storage
 test -e /etc/sysconfig/crio-network && source /etc/sysconfig/crio-network
 
-exec /usr/bin/crio --log-level=$LOG_LEVEL
+exec /usr/bin/crio --log-level=$LOG_LEVEL --enable-metrics --metrics-port 4242
```

Comment:
Why not use the default port (9090)?  Also, both `--enable-metrics` and `--metrics-port` look like they need man entries.

Reply:
fine by me if you want to hard code it, another option is to use `$METRICS_PORT`, you need to change `manifest.json` to assign it a default value and add it to the env block in the `config.json.template` file, something like:

```
diff --git a/contrib/system_containers/centos/config.json.template b/contrib/system_containers/centos/config.json.template
index 785383d4..57171c14 100644
--- a/contrib/system_containers/centos/config.json.template
+++ b/contrib/system_containers/centos/config.json.template
@@ -208,6 +208,7 @@
         "selinuxLabel": "system_u:system_r:container_runtime_t:s0",
         "cwd": "/",
         "env": [
+            "METRICS_PORT=$METRICS_PORT",
             "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/root/go/bin",
             "TERM=xterm",
             "LOG_LEVEL=$LOG_LEVEL",

diff --git a/contrib/system_containers/centos/manifest.json b/contrib/system_containers/centos/manifest.json
index 38f4dc87..14f3194f 100644
--- a/contrib/system_containers/centos/manifest.json
+++ b/contrib/system_containers/centos/manifest.json
@@ -1,6 +1,7 @@
 {
     "version": "1.0",
     "defaultValues": {
+        "METRICS_PORT" : "4242",
         "LOG_LEVEL" : "info",
         "OPT_CNI" : "/opt/cni",
         "VAR_LIB_CONTAINERS_STORAGE" : "/var/lib/containers/storage",
```

Once we have this in place, it will be trivial to customize these settings from the installer.